### PR TITLE
fix: use node copyFile for cross-platform support

### LINKED ISSUE #48712: NextJS examples/cms-sanity > npm run studio:dev error
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Home
    Binaries:
      Node: 18.16.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.3.1
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
```

### Which example does this report relate to?

cms-sanity

### What browser are you using? (if relevant)

Chrome Version 112.0.5615.138

### How are you deploying your application? (if relevant)

_No response_

### Describe the Bug

Visiting https://github.com/vercel/next.js/tree/canary/examples/cms-sanity I've clicked Deploy under Step 1.

I've created the github repo, created a sanitycms project, connected them both to Vercel, cloned the github repo locally, then followed instructions in the local README; run `npx vercel link` fine, run `npx vercel env pull` fine, run `yarn install && yarn dev` fine.

In a new terminal tab, when I run `npm run studio:dev` I receive an error:

```
> prestart
> npm run env

> env
> cp ../.env .env.development || cp ../.env.local .env.development

'cp' is not recognized as an internal or external command,
operable program or batch file.
'cp' is not recognized as an internal or external command,
operable program or batch file.
```

### Expected Behavior

To start the Sanity Studio at https://localhost:3333 through `npm run studio:dev` without an error.

### To Reproduce

Clone the repo, follow steps 1 to 4. At Step 4, check the error.

### PR DESCRIPTION
**Pull request by darshkpatel**
Fixes: #48712

Use node copyFile in the package.json script for cross-platform support

### CODE REVIEW

File: examples/cms-sanity/studio/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "deploy": "sanity deploy",
     "prestart": "npm run env",
     "predeploy": "npm run env",
-    "env": "cp ../.env .env.development || cp ../.env.local .env.development"
+    "env": "shx cp ../.env .env.development || shx cp ../.env.local .env.development"
```
Comment (styfle) [inline examples/cms-sanity/studio/package.json:11]: Lets avoid adding new dependencies.

Instead, use an inline node script:

https://nodejs.org/api/fs.html#fspromisescopyfilesrc-dest-mode

Comment (darshkpatel) [reply examples/cms-sanity/studio/package.json]: Done!

**Review by styfle — DISMISSED** [review]

File: examples/cms-sanity/studio/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "deploy": "sanity deploy",
     "prestart": "npm run env",
     "predeploy": "npm run env",
-    "env": "cp ../.env .env.development || cp ../.env.local .env.development"
+    "env": "node -e \"const fs=require('fs');fs.existsSync('../.env')?fs.copyFileSync('../.env'), '.env.development'):fs.copyFileSync('../.env.local','.env.development');\""
```
Comment (ijjk) [inline examples/cms-sanity/studio/package.json:11]: Feels like this is prone to errors, could we make it a JS file instead that is invoked here?

Comment (darshkpatel) [reply examples/cms-sanity/studio/package.json]: Refactored! Apologies for the delay, forgot about the PR : /

**Review by ijjk — APPROVED** [review]

**Review by styfle — APPROVED** [review]

**Pull request merged**